### PR TITLE
blockdev_stream_filter_nodename:Block stream with filter-node-name set

### PR DESCRIPTION
**Pull request by aliang123**
id:1988899

### CODE REVIEW

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.blockdev_stream_filter_nodename.q35: PASS (41.17 s)

**Comment by aliang123** [comment]
@zhencliu @ngu-niny, please help to review, thanks.

File: qemu/tests/blockdev_stream_filter_nodename.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    def do_test(self):
+        self.snapshot_test()
+        self.blockdev_stream()
+        job_utils.check_block_jobs_started(
+            self.main_vm, [self._job],
+            self.params.get_numeric('job_started_timeout', 60)
+        )
+        self.check_filter_nodes_name()
+        self.wait_stream_job_completed()
+
```
Comment (ngu-niny) [inline qemu/tests/blockdev_stream_filter_nodename.py:39]: Better to check block info again to make sure there is no 'filter_node_name' after the stream job finished.

File: qemu/tests/blockdev_stream_filter_nodename.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+class BlkdevStreamFilterNode(blockdev_stream_nowait.BlockdevStreamNowaitTest):
+    """
+    blockdev-stream filter node name test
+    """
+
+    def _init_stream_options(self):
+        super(BlkdevStreamFilterNode, self)._init_stream_options()
+        filter_node_name = self.params.get("filter_node_name")
+        if filter_node_name:
+            self._stream_options["filter-node-name"] = filter_node_name
```
Comment (zhencliu) [inline qemu/tests/blockdev_stream_filter_nodename.py:14]: self._stream_options["filter-node-name"] = self.params['filter_node_name']
for this case is designed for filter node test, filter_node_name param is a must.
But yours is still OK for me

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.blockdev_stream_filter_nodename.q35: PASS (41.81 s)

File: qemu/tests/blockdev_stream_filter_nodename.py (hunk 1 above)
Comment (aliang123) [reply qemu/tests/blockdev_stream_filter_nodename.py:50]: Have updated, please help to review again, thanks

File: qemu/tests/blockdev_stream_filter_nodename.py (hunk 2 above)
Comment (aliang123) [reply qemu/tests/blockdev_stream_filter_nodename.py]: @zhencliu  updated, thanks.

**Comment by ngu-niny** [comment]
1-Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.ppc64le.io-github-autotest-qemu.blockdev_stream_filter_nodename Finshed 2021-08-06 00:25:27 2021-08-06 00:27:43 PASS 135.656819
2-Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_stream_filter_nodename Finshed 2021-08-06 00:27:43 2021-08-06 00:30:19 PASS 156.488898

Acked-by: Nini Gu <ngu@redhat.com>

**Comment by zhencliu** [comment]
LGTM

**Pull request merged**